pkg/storage/unified/resource/search.go

```diff
@@ -4,9 +4,12 @@ import (
 	"context"
 	"fmt"
 	"log/slog"
+	"slices"
 	"sync"
 	"time"
 
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
+	"github.com/grafana/grafana/pkg/storage/unified/search"
```

Comment:
we need to avoid the circular dependency here -- `unified/search` can depend on `unified/resource` but not the other way around